lv_lgia/lv_14_convert.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Ensure an input file is provided
+if [ -z "$1" ]; then
+    echo "Usage: $0 <path_to_source_file>"
+    exit 1
+fi
+
+# Get the absolute path of the input file
+SOURCE_FILE="$1"
+
+# Ensure the input file exists
+if [ ! -f "$SOURCE_FILE" ]; then
+    echo "Error: File '$SOURCE_FILE' not found."
+    exit 1
+fi
+
+# Define a temporary file for the corrected grid
+TEMP_FILE="temp_grid.dat"
+
+# Swap latitude and longitude using awk and create a temporary file
+awk '{print $2, $1, $3}' "$SOURCE_FILE" > "$TEMP_FILE"
+
+# Convert the corrected grid to a GeoTIFF
+OUTPUT_TIF="lv_lgia_lv14.tif"
+gdal_translate -of GTiff -a_nodata 9999 "$TEMP_FILE" "$OUTPUT_TIF"
+
+# Run the vertoffset grid conversion
+../grid_tools/vertoffset_grid_to_gtiff.py \
+  --description "LKS92 (EPSG:4949) to Latvia 2000 height (EPSG:7700). Converted from $SOURCE_FILE" \
+  --type "GEOGRAPHIC_TO_VERTICAL" \
+  --copyright "Derived from work by Latvian Geospatial Information Agency. CC-BY 4.0" \
+  --area-of-use "Latvia - onshore mainland" \
+  --parameter-name "geoid_undulation" \
+  --source-crs "EPSG:4949" \
+  --target-crs "EPSG:7700" \
+  "$OUTPUT_TIF" "$OUTPUT_TIF"
```

Comment:
Using the same name for input and output is not a good idea. Just use another temp file.